soda/core/soda/scan.py

```diff
@@ -370,14 +370,15 @@ def execute(self):
             fail_text = "failure" if checks_warn_count == 1 else "failures"
             error_count = len(self.get_error_logs())
             error_text = "error" if error_count == 1 else "errors"
-            self.__log_checks(None)
```

Comment:
This line logs all the checks that are `NOT EVALUATED`. This may happen when there are problems computing the metrics it depends on.  

I think it's important that the summary explicitely logs which checks have not been evaluated.  Is that done in another way?  Or in other words, why do you think this line can be removed?

Reply:
I misunderstood what the method did and did not realize it actually produced output to logs in such case, I thought it was a forgotten line. Thanks.